packages/vite/src/node/build.ts

```diff
@@ -180,10 +181,10 @@ export interface BuildOptions {
    */
   chunkSizeWarningLimit?: number
   /**
-   * Rollup watch options
-   * https://rollupjs.org/guide/en/#watchoptions
+   * Chokidar watch options
+   * https://github.com/paulmillr/chokidar
    */
-  watch?: WatcherOptions | null
+  watch?: WatcherOptions | WatchOptions | null // WatcherOptions is deprecated
```

Comment:
Looks like the null value is implicit by making `watch` optional. 
```suggestion
  watch?: WatcherOptions | WatchOptions | boolean // WatcherOptions is deprecated
```
But we could add the possibility to use `watch: true` to get the default watching options, that is what the majority of users would use, no?

Reply:
> Looks like the null value is implicit by making watch optional.
Hum, null is not the same as optional (undefined) but yes boolean is better here

Reply:
Just had a quick look, but we use mostly just `false` instead of `boolean` cause `true` is not an option but it should be `WatcherOptions | WatchOptions` if not `false`, should it?

Reply:
But we want to allow for `watch: true` here, if not users are forced to write `watch: {}`

Reply:
Ok